### PR TITLE
Fixes #16991 - Allow setting ssh log level

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by adamruzicka** [comment]
@iNecas Could you please take a look at this, the accompanying PR in REX[1] was already merged some time ago.

[1] - https://github.com/theforeman/foreman_remote_execution/pull/290

File: settings.d/remote_execution_ssh.yml.example [hunk 1]
```diff
@@ -10,3 +10,7 @@
 # for new data leave empty to use the runner's default
 # (1 second for regular, 60 seconds with async_ssh enabled)
 # :runner_refresh_interval:
+
+# Defines the verbosity of logging coming from Net::SSH
+# one of :debug, :info, :warn, :error, :fatal
```
Comment (iNecas) [inline settings.d/remote_execution_ssh.yml.example]: If I remember properly, the `ssh_log_level` should be higher than the `smart_proxy_dynflow_core` log level. Perhaps we should state it here as well

**Comment by adamruzicka** [comment]
Updated

**Comment by iNecas** [comment]
Thanks @adamruzicka

**Pull request merged**